src/home_robot/home_robot/agent/ovmm_agent/llm_agent.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import json
+from enum import IntEnum, auto
+from typing import Any, Dict, Optional, Tuple
+
+import clip
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+from PIL import Image
+
+from home_robot.agent.objectnav_agent.objectnav_agent import ObjectNavAgent
+from home_robot.agent.ovmm_agent.ovmm_agent import OpenVocabManipAgent
+from home_robot.agent.ovmm_agent.ppo_agent import PPOAgent
+from home_robot.core.interfaces import DiscreteNavigationAction, Observations
+from home_robot.manipulation import HeuristicPlacePolicy
+
+
+class Skill(IntEnum):
+    NAV_TO_OBJ = auto()
+    GAZE_AT_OBJ = auto()
+    PICK = auto()
+    NAV_TO_REC = auto()
+    GAZE_AT_REC = auto()
+    PLACE = auto()
+
+
+def get_skill_as_one_hot_dict(curr_skill: Skill):
+    skill_dict = {skill.name: 0 for skill in Skill}
+    skill_dict[f"is_curr_skill_{Skill(curr_skill).name}"] = 1
+    return skill_dict
+
+
+class LLMAgent(OpenVocabManipAgent):
+    """Simple object nav agent based on a 2D semantic map."""
+
+    def __init__(self, config, device_id: int = 0, obs_spaces=None, action_spaces=None):
+        super().__init__(config, device_id=device_id)
```

Comment:
Add a "warning" here to say this is not fully supported yet, when you create it

Reply:
Fixed